fhttp/httprunner.go

```diff
@@ -193,14 +200,27 @@ func RunHTTPTest(o *HTTPRunnerOptions) (*HTTPRunnerResults, error) {
 		total.sizes.Transfer(httpstate[i].sizes)
 		total.headerSizes.Transfer(httpstate[i].headerSizes)
 	}
+
+	// Sort the ip address form largest to smallest based on its usage count
+	ipList := make([]string, 0, len(total.IPCountMap))
+	for k := range total.IPCountMap {
+		ipList = append(ipList, k)
```

Comment:
it's somewhat minor but you should be able to get that list in the above loop; like `keys` - though maybe it's easier/as clear as you have it now; given we don't expect hundreds of ips anyway